README.md

```diff
@@ -1,15 +1,17 @@
 ~~~~
-  _____ _______ _____   _   _   _____   
- / ____|__   __|_   _| | \ | | / ____|  
-| (___    | |    | |   |  \| || |  __   
- \___ \   | |    | |   | . ` || | |_ |  
- ____) |  | | _ _| |_ _| |\  || |__| |_
-|_____(_) |_|(_)_____(_)_| \_(_)_____(_)
-
-  Sturdy Tactical Integrated Nerf Gun
+  ___   _____ _____  ______ _       ___   _____ _____ ___________
```

Comment:
Minor styling quibbles:
* Please use the default font from `figlet`, to match the original style.
* Also (arguably), since this is an acronym, there should be periods.